skycultures/indian/constellation_names.eng.fab

```diff
@@ -19,49 +20,53 @@
-# TRANSLATORS: āṣāṛā - Roar
-N20	"Pūrba āṣāṛā"		_("The former undefeated one")
-# TRANSLATORS: āṣāṛā - Roar
-N21	"Uttara āṣāṛā"		_("The latter undefeated one")
-N22	"Śrabaṇā"		_("The Ear")
-N23	"Dhaniṣṭhā"		_("The most famous one")
-N24	"Śatabhiṣā"		_("hundred cures")
-N25	"Pūrba bhādrapada"	_("The first of the blessed feet")
-N26	"Uttara bhādrapada"	_("The second of the blessed feet")
-N27	"Revatī"		_("The prosperous one")
-N28	"Abhijit"		_("The invincible one")
+# Provenance for art - BG Tilak's book, veda-s (eg. Aitareya AraNyaka quoted elsewhere in this doc). 
+mRgashiras	"Mṛgaśiras"	_("Deer's head")
+# Associated with the snakes, as per taittirIya bhrAhmaNa.
+AshleSha	"Aśleṣa"	_("Embrace")
+hasta	"Hasta"	_("Hand")
+mUla	"Mula"	_("Root")
+shravaNa	"Śravaṇa"	_("Ear")
+pUrvabhAdrapada	"Pūrva bhādrapada"	_("The first of the blessed feet")
+uttarabhAdrapada	"Uttara bhādrapada"	_("The second of the blessed feet")
+#################
+# Some Rāśis
+#################
+# For simplicity in mapping prexisting constellation art, we use some codes from the western skyculture.
+# Where the latin and sanskrit match, we follow the instruction from the manual: "please use the canonical Latin names, they have all been translated already."
+Ari	"Meṣa Rāśi"	_("Aries")
+Tau	"Vṛṣa Rāśi"	_("Taurus")
+Gem	"Mithuna Rāśi"	_("Gemini")
+Cnc	"Karkaṭa Rāśi"	_("Cancer")
+Leo	"Simha Rāśi"	_("Leo")
+Vir	"Kanyā Rāśi"	_("Virgo")
+Lib	"Tulā Rāśi"	_("Libra")
+Sco	"Vṛścikā Rāśi"	_("Scorpius")
+Sgr	"Dhanur Rāśi"	_("Bow")
+makara	"Makara Rāśi"	_("Crocodile")
+Aqr	"Kumbha Rāśi"	_("Aquarius")
+Psc	"Mīna Rāśi"	_("Pisces")
+#################
+# Other constellations.
+#################
+# Please include provenance citations for the constellations you add.
+# Where the latin and sanskrit match, we follow the instruction from the manual: "please use the canonical Latin names, they have all been translated already."
+# Provenance for the below: Aitareya AryaNyaka: तम् अभ्यायत्याविध्यत्, स विद्ध ऊर्ध्व उदप्र(प्ल)वत । तम् एतम् मृग इत्य् आचक्षते । य उ एव मृगव्याधः स उ एव स, या रोहित् सा रोहिणी, यो एवेषुस् त्रिकाण्डा स एवेषुस् त्रिकाण्डा । तद् वा इदम् प्रजापते रेतः सिक्तम् अधावत्, तत् सरो ऽभवत् (=आकाशगङ्गा)।
+mRgavyAdha	"Mṛgavyādha"	_("Deer Hunter")
+# Provenance for the below: TaittirIya AraNyakam - https://archive.org/stream/Anandashram_Samskrita_Granthavali_Anandashram_Sanskrit_Series/ASS_036_Taittiriya_Aranyakam_With_Sayana_Bhashya_Part_1_-_Babasastri_Phadke_1898#page/n161/mode/2up
+shimshumAra	"Shiṁśumāra"	_("Dolphin")
+# Provenance for the below: Apte dictionary, purana-s.
+UMa	"ṛkṣa"	_("Ursa Major")
+# Provenance - BG Tilak's book, veda-s (eg. Aitareya AraNyaka quoted above). Amarakosha says:  मृगशीर्षं मृगशिरस्तस्मिन्नेवाग्रहायणी। इल्वलास् तच्छिरोदेशे तारका निवसन्ति याः॥. Some modern dictionaries thence identify iNvakA-s with the head of Orion, but given its identification with the arrow, it is more likely to be Orion's belt as illustrated by Tilak. 
+iNvakA	"Invakā"	_("Arrow")
```

Comment:
Need a put an EOL char (add new empty line)

Reply:
Not only that. Please add at least approximate English translations to the Indian strings. You can explicitly state [approximately: ... translation: vvasuki]. 
For the provenance, a file references.fab (see chapter 9.11 in the User Guide) may be helpful.

The Indian skyculture is highly complex, and Stellarium currently is not able to map this complexity. I have the impression this is the third iteration of additions I am aware of, every time bringing new spelling variants to translate, however often without new translatable (understandable) content. 

What I dislike currently is a separation of the Nakshatras into asterisms or constellations "based on available sky art". This does not make any sense to me. If nakshatras deserve sky art in principle, they should all be defined as constellations, and sky art can be added also later. You can comment them as nakshatras so that when the time comes (see below), they will be separated out. The distinction into asterisms vs. constellations is a modern concept of "adding line art bits in addition to the canonical 88 constellations" that has been developed and works best in context of the Western skyculture.  
If needed, you can repeat all line art for nakshatras again into the asterisms file to allow "display asterisms==display nakshatras". Then of course don't add other asterisms to that file. 

We are in these months discussing extending the whole skyculture thing, but have not defined any new formats yet. I would like to make Lunar Stations and Zodiacal constellations two distinct groups which both should be displayable also with artwork. And of course, single-star "constellations" should be marked in some way when lines are active (just a circled star?).  But there are also still astronomical problems to solve. We just don't have enough developers.

Reply:
> Need a put an EOL char (add new empty line)

Done.

> Not only that. Please add at least approximate English translations to the Indian strings. You can explicitly state [approximately: ... translation: vvasuki].

I checked and found approximate english translations for constellation and star names. Was there something else you needed translated?

> For the provenance, a file references.fab (see chapter 9.11 in the User Guide) may be helpful.

Good idea - added. I feel it is important to **allow comments in each .fab file** - that way one can add line-specific comments. Currently, one cannot provide citation for a particular line-art or sky-art.

>  I have the impression this is the third iteration of additions I am aware of, every time bringing new spelling variants to translate, however often without new translatable (understandable) content.

Part of the problem was that a contributor (Tanmoy) comes from a region of India (Bengal) where they don't distinguish between v and b; and he did not show sufficient Sanskrit language and transliteration knowledge to accurately type the names. That's the only source of variation - otherwise, there is no dispute about the canonical transliterated spelling. 
I also don't fully understand the comment - could you please provide an example for " without new translatable (understandable) content" here?

> What I dislike currently is a separation of the Nakshatras into asterisms or constellations "based on available sky art". This does not make any sense to me. If nakshatras deserve sky art in principle, they should all be defined as constellations, and sky art can be added also later.

To clarify, *some* nakShatra-s deserve sky art (png files, rather than lines). Not all. It is not so much a question of availability as much as *existence*. Given this clarification, how do you recommend we proceed?

Reply:
We are in a process of collecting and gathering requirements for better handling of skycultures, although we fundamentally lack development time (and, I openly admit, ethnological and cultural knowledge). This includes a coming separation of Lunar stations and probably Zodiacal constellations. This will not happen next week, but - time allowing - within the next few years. 

"Asterisms" were added in the last years to show what "modern" people understand as "inofficial" star group, like the Big Dipper, Summer Triangle or Winter Hexagon. Those figures currently don't show constellation art, although clearly a Big Dipper or Sickle could be shown with an artistic rendering, so this may also change, if somebody is willing to add them. Given the uncontrolled growth of asterisms, we should also find some way of selecting favourites, e.g. there are "Winter Hexagon" and "Big G" formed by practically the same stars, seeing any of the two is a personal preference.  

If inside a collection of entities there are entries with some common optional peculiarity, all entities should go into the class that allows such peculiarity. In our case, if *some* Nakshatras may receive sky art, *all* Nakshatras should be coded as constellations. When we do the change mentioned above, those will then be separated (or just marked inside the constellation file). I don't know whether the Indian traditional sky has non-constellational asterisms in a similar add-on concept as described above. If you want to have a way to "show all nakshatras as line art, while leaving the other constellations switched off", add all nakshatras additionally as the only possible asterisms. Then you can, for now, switch on "asterisms" to show the "nakshatras". 

Currently we have no way to show a 1-star constellation/asterism. A workaround would be a very short line to the closest (also dim!) HIP star, so that at least a label gets placed next to the star, and write in the comments that the dim star is not official part of that constellation. We may highlight the 1-star constellation in later editions e.g. by a circle. 

We should indeed be able to add comment handling (just ignore everything past a `#` sign) very soon. After the comment sign, we may collect extra data per line, but have not decided about the formats yet. You see the listing of references in the star_names.fab. We may use a similar thing for the other lists. So, the first string after the `#` can be a comma-separated list of references.

On "without translatable (understandable) content": e.g.
> pUrvAShADhA	_("Pūrva aṣādhā", "asterism")
> uttarAShADhA	_("Uttara aṣādhā", "asterism")

Does Asadha mean anything? Or Purva and Uttara? 

You are using full (transcribed) names as abbreviation for the constellations. The "abbreviated" mode should allow to use short unique markers, without filling the screen with text. In scientific context, the 88 IAU constellations are abbreviated with 3 characters, which is just enough to remember the constellation. I am sure you can find such shorter names. The old codes N01, N02...  were also OK because they marked the nakshatras. 

You deleted the month names from the description. Were they wrong?